docs/guides/python-flask-abtests.mdx

```diff
@@ -0,0 +1,112 @@
+---
+sidebar_label: Python/Flask A/B Test
+title: Python/Flask A/B Test
+---
+
+Flask is a popular lightweight framework for web development with Python. While Flask is known for its simplicity and ease-of-use, its also powerful enough to use at scale for a consumer-facing application, where you might find yourself wanting to run a simple A/B Test. Today, we'll show you how to do just that with Statsig's free tier!
+
+Today we'll use our [Python Server SDK](https://docs.statsig.com/server/pythonSDK) - which works well with Flask out-of-the-box given Flask's limited abstraction on top of Python. Our goal today is to create a website path or "route" in Flask that returns different results based on who the user_id - effectively creating an A/B test. 
+
+## Step 1: Setup
+
+First, we need to create a directory and a file to work in. 
+
+```zsh
+mkdir Flask-Flags
+cd Flask-Flags
+touch app.py
+```
+
+app.py is the file we'll write all of our code in. Next, we should install the Flask and Statsig Python packages if you haven't already. Note - Statsig requires you to use Python 3, for more information take a look at our [Python SDK page.](https://docs.statsig.com/server/pythonSDK)
+
+```
+pip3 install Statsig, Flask
+```
+
+## Step 2: Setting up Flask
+Next we'll start editing app.py - we'd recommend opening it up in your favourite code editor to get started. If you're new at this, you can open it in any text editor. We'll start with the simplest possible, 6-line flask app. 
+
+```Python 
+from flask import Flask
+
+app = Flask(__name__)
+
+@app.route('/')
+def index():
+    return f"Hello World!"
```

Comment:
I think if you're not referencing any variables in the string with {variable} we should not use the f

Reply:
Thank you, actioned!